templates/unit.erb

```diff
@@ -0,0 +1,38 @@
+[Unit]
+Description=Apache Kafka server (<%= (@service_name.split(/-/)[1] and @service_name.split(/-/)[1].capitalize) or 'broker' -%>)
+Documentation=http://kafka.apache.org/documentation.html
+Wants=zookeeper.service
+After=zookeeper.service
+Requires=zookeeper.service
```

Comment:
most probably. but for now that's what our upstream module calls it: https://github.com/deric/puppet-zookeeper/blob/master/manifests/service.pp#L11

n.b.: our upstream module does the same bad thing to systemd.